test/mod/test_monitor.py

```diff
@@ -18,14 +18,13 @@
 from .. import test
 
 
-def echo(path):
+def echo():
     """echo stdin to stdout after setting stdio up via API
 
     Meant to be called as the main function in a process
     simulating an osbuild runner and a stage run which does
     nothing but returns the supplied options to stdout again.
     """
-    osbuild.api.setup_stdio(path)
     data = json.load(sys.stdin)
     json.dump(data, sys.stdout)
     sys.exit(0)
```

Comment:
```
test_log_monitor_api (test.mod.test_monitor.TestMonitor) ... Process Process-2:
Traceback (most recent call last):
  File "/usr/lib64/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib64/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
TypeError: echo() takes 0 positional arguments but 1 was given
FAIL
```

Reply:
The caller of `echo()` needs to be adjusted as well. It still passes the path as argument.